.github/workflows/workflow_manage_public_access_ur_environment.yml

```diff
@@ -0,0 +1,39 @@
+name: "Toggle public access to UR environment"
+
+on:
+  workflow_dispatch:
+    inputs:
+      public_access_enabled:
+        description: 'Enable public access to the UR environment?'
+        required: true
+        type: boolean
+
+permissions:
+  id-token: write
+  contents: write
+  security-events: write
+  pull-requests: write
+  actions: none
+  checks: none
+  deployments: none
+  issues: write
+  packages: none
+  repository-projects: none
+  statuses: none
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  ur_toggle_public_access:
+      name: UR Environment Deploy
+      needs: [ui_tests_image]
+      uses: ./.github/workflows/toggle_public_access_job.yml
+      with:
+        workspace_name: 411mlpab799
```

Comment:
should it be this?

Reply:
yes, while I'm testing I don't want to modify the ur env (it's also in use at the moment). While I've tested the commands etc. I also can't access this workflow dispatch while it's in the PR. Once it's merged I'll be able to access the job (and branch versions of it too)

Reply:
yes, I don't want to modify UR while it's in use. While this is a PR, the workflow dispatch job doesn't appear for me to trigger and test (I've tested the commands etc).